core/src/main/java/apoc/ApocConfig.java

```diff
@@ -57,7 +57,9 @@
     public static final String APOC_CONFIG_JOBS_SCHEDULED_NUM_THREADS = "apoc.jobs.scheduled.num_threads";
     public static final String APOC_CONFIG_JOBS_POOL_NUM_THREADS = "apoc.jobs.pool.num_threads";
     public static final String APOC_CONFIG_JOBS_QUEUE_SIZE = "apoc.jobs.queue.size";
-    public static final String APOC_CONFIG_INITIALIZER_CYPHER = "apoc.initializer.cypher";
+    public static final String APOC_CONFIG_INITIALIZER = "apoc.initializer";
+    @Deprecated
```

Comment:
It's nice if you're deprecating things to add some info on what the replacement is:
e.g.
```
/**
     * @deprecated
     * This has been replaced by database-specific initialisers.
     * Use apoc.initializer.<database name> instead.
     */
```

Reply:
good catch, added your suggestions.